Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -33,6 +33,9 @@ open class MapView: UIView {
     /// Controls the addition/removal of annotations to the map.
     public internal(set) var annotations: AnnotationOrchestrator!
 
+    /// Controls the display of attribution dialogs
+    internal var attributionDialogManager: AttributionDialogManager!
```

Comment:
Could this be private if we moved the extension into this file?

Reply:
I also had a question about this, Above I was wondering if this can move out of the mapView

Reply:
Testing :/

Reply:
Not sure exactly what you mean, but can you inject this using the constructor that's for testing?